modules/kubernetes-addons/external-dns/values.yaml

```diff
@@ -1,4 +1,3 @@
 provider: aws
-zoneIdFilters: ${zone_filter_ids}
```

Comment:
do we know what was the original reason for having `zoneIdFilters` set? can we safely remove it? checking https://github.com/bitnami/charts/blob/master/bitnami/external-dns/values.yaml#L609-L611 I am not sure, does external DNS know automatically which zone id to use? iirc domain filter is what used mainly but I might be wrong

Reply:
yes, I looked this up as well and it all looks to be optional. so to provide better flexibility, I think defaulting to the minimum required values and letting users opt in to additional values will scale better (they now can choose to set zone ID filters, zone name filters, domain name filters, etc. or not)